.github/workflows/licensing.yml

```diff
@@ -0,0 +1,26 @@
+name: Licensing Compliance
+
+on:
+  push:
+    branches: [ trunk, develop ]
+  pull_request:
+    branches: [ trunk, develop ]
+  workflow_dispatch:
+
+jobs:
+  licensing:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Advance Security Compliance Action
+      uses: GeekMasher/advanced-security-compliance@v1.6
+      with:
+        # We'll need an access token that can call the GraphQL GitHub API to get
+        #  all the of the licensing data.
+        token: ${{ secrets.ACCESS_TOKEN }}
+        # Disable everything else to check for and focus on licensing
+        argvs: '--disable-dependabot --disable-secret-scanning --disable-code-scanning'
+        # The owner/repo of where the policy is stored  
+        policy: 10up/.github
+        # The local (within the workspace) or repository
+        policy-path: blob/trunk/policy.yml
```

Comment:
@jeffpaul This policy path, is there sub folders called "blob/truck"? The tool does not need this and is relative to the base path in the repo.

Example:

```yml
- name: Advance Security Compliance Action
  uses: GeekMasher/advanced-security-compliance@v1.6.1
  with:
    # The owner/repo of where the policy is stored  
    policy: GeekMasher/security-queries
    # The local (within the workspace) or repository
    policy-path: policies/default.yml
    # The branch you want to target
    policy-branch: main
```

Points to this file https://github.com/GeekMasher/security-queries/blob/main/policies/default.yml